Formula/p/pawk.rb

```diff
@@ -0,0 +1,27 @@
+class Pawk < Formula
+  include Language::Python::Shebang
+
+  desc "Python line processor (like AWK)"
+  homepage "https://github.com/alecthomas/pawk"
+  url "https://github.com/alecthomas/pawk/archive/refs/tags/v0.8.1.tar.gz"
+  sha256 "727955d22f0f88ed06443fda067d67d637373cb7dd9debf01ffa3db5b8f907f4"
```

Comment:
let's use pypi source rather than github one

```suggestion
  url "https://files.pythonhosted.org/packages/6c/90/2165e9fedd33ac172899aa3df6754971d720bf07eef2a0b049db15a7ad69/pawk-0.8.1.tar.gz"
  sha256 "59ec1a4046cf545e1376c8c0a28f5f178a3b88dbc85fb3772aa3ce8c2e088349"
```

Reply:
Does `livecheck` work for PyPI sources as well? Because that was a one off issue, upstream swiftly resolved it as soon as I brought it up to their attention. The fix in the that version was PyPI specific, and probably that's why tag creation was missed.

Reply:
yeah, livecheck would handle pypi source as well.

Reply:
@manakbisht can you make the source url change, I think we are almost there.